### ISSUE TITLE
Add performance level cutoffs to MCAS score graph

### ISSUE THREAD

**Issue by marimuraki** [codeforamerica/somerville-teacher-tool#172]
Teachers said that it would be helpful to have the proficiency cutoffs on the MCAS scale score graphs.

Performance Level	 | Scaled Score Point Range
------------ | -------------
Advanced | 260–280
Proficient | 240–258
Needs Improvement | 220–238
Warning (grades 3–8), Failing (HS) | 200–218

**Comment by marimuraki** [comment]
**Design Priority** for @AmirHadjihabib 
- mock up how adding proficiency cutoffs in the MCAS scale score graph would look like :chart_with_upwards_trend:

**Comment by alexsoble** [comment]
@marimuraki — So the min score a student can achieve would be 200 and the max score 280, correct?

**Comment by marimuraki** [comment]
@alexsoble: Correct. 

I'll do a quick check for outliers due to any input errors.

**Comment by marimuraki** [comment]
@alexsoble: Exclude MCAS scale score == 0 (invalid values).

After excluding MCAS scale scores == 0, we observe in the latest X2 export:

- min: 202
- max: 342

@alexsoble: We need to look into out-of-range values < 200 and > 280. Let me dig deeper and see what I find.

**Comment by alexsoble** [comment]
@marimuraki :+1: :+1:

**Comment by marimuraki** [comment]
Only 1 obs in latest X2 export with out of bounds MCAS scale scores (=342 >> 280 max). Can assume such values incorrect (replace to missing).

**Comment by marimuraki** [comment]
@alexsoble: @AmirHadjihabib and I mocked up what it would look like. What do you think?

![profile - landing - mcas](https://cloud.githubusercontent.com/assets/1334815/8788358/af2fe762-2eee-11e5-8681-e5522695bcfa.png)

**Comment by marimuraki** [comment]
**Feedback from Vince & principals on MCAS Growth graph**

Currently benchmark at 40th percentile & text incorrectly says "points"

- Benchmark line on graph
  - Put line for typical growth at 50th percentile or range between 40th - 60th percentile
  - Don't want parents or teachers to think that anything above a 40th percentile is good.

**Comment by marimuraki** [comment]
@AmirHadjihabib: Let me know what you think. We can work together on making it look prettier. Maybe put text on lower line below rather than on top?

<img width="747" alt="screenshot 2015-08-12 14 23 48" src="https://cloud.githubusercontent.com/assets/1334815/9237665/3f96db66-4101-11e5-84b4-6b0e9501d149.png">

<img width="749" alt="screenshot 2015-08-12 14 24 00" src="https://cloud.githubusercontent.com/assets/1334815/9237670/4554e21e-4101-11e5-94e5-cf09e5c5e637.png">

**Comment by marimuraki** [comment]
@AmirHadjihabib @alexsoble: I updated the MCAS graphs to have the MCAS scale score to performance level bands. 

@AmirHadjihabib: Feel free to change the colors as necessary. I tried the grey and white colors like in the mock-up but I didn't like how it made the Warning and Proficient pop out more than the Needs Improvement and Advanced levels --- when all levels are just as important. Visually it was not signalling that all levels.

*BEFORE*
<img width="529" alt="screenshot 2015-08-12 17 30 18" src="https://cloud.githubusercontent.com/assets/1334815/9240260/35e781d4-4119-11e5-9add-fe629ca34e2a.png">

*NEW*
<img width="527" alt="screenshot 2015-08-12 17 37 54" src="https://cloud.githubusercontent.com/assets/1334815/9240288/4277bea0-4119-11e5-97af-de131e0fce5c.png">
